model/task/task.go

```diff
@@ -2334,6 +2334,9 @@ func (t *Task) findDisplayStatus() string {
 	if t.Status == evergreen.TaskSucceeded {
 		return evergreen.TaskSucceeded
 	}
+	if t.HasAnnotations {
+		return evergreen.TaskKnownIssue
+	}
```

Comment:
I believe this should be equivalent to the [agg pipeline](https://github.com/evergreen-ci/evergreen/blob/30b1913586d8e0bbf2862a41231a6aae14f0f039/model/task/db.go#L215-L218) which has Known Issue as the status of highest priority